src/statuspage.ts

```diff
@@ -0,0 +1,27 @@
+import DefaultOperations from './defaultOperations';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/zipper-team/statuspage/pullRequest?prid=7096534)
```suggestion
import DefaultOperations from "./defaultOperations";
```